DSharpPlus.Rest/DiscordRestClient.cs

```diff
@@ -369,10 +369,11 @@ public Task<DiscordChannel> CreateGuildChannelAsync(ulong id, string name, Chann
         /// <param name="bitrate">New voice channel bitrate</param>
         /// <param name="userLimit">New voice channel user limit</param>
         /// <param name="perUserRateLimit">Slow mode timeout for users.</param>
+        /// <param name="rtcRegion">New region override.</param>
         /// <param name="reason">Reason why this channel was modified</param>
         /// <returns></returns>
-        public Task ModifyChannelAsync(ulong id, string name, int? position, Optional<string> topic, bool? nsfw, Optional<ulong?> parent, int? bitrate, int? userLimit, Optional<int?> perUserRateLimit, string reason)
-            => this.ApiClient.ModifyChannelAsync(id, name, position, topic, nsfw, parent, bitrate, userLimit, perUserRateLimit, reason);
+        public Task ModifyChannelAsync(ulong id, string name, int? position, Optional<string> topic, bool? nsfw, Optional<ulong?> parent, int? bitrate, int? userLimit, Optional<int?> perUserRateLimit, Optional<DiscordVoiceRegion> rtcRegion, string reason)
+            => this.ApiClient.ModifyChannelAsync(id, name, position, topic, nsfw, parent, bitrate, userLimit, perUserRateLimit, rtcRegion != null ? rtcRegion.IfPresent(e => e.Id) : null, reason);
```

Comment:
I think `rtcRegion != null ? rtcRegion.IfPresent(e => e.Id) : null` can be changed to `rtcRegion?.IfPresent(e => e.Id)`

Reply:
You can't use `?` on `Optional` it looks like

Reply:
Right, I misread. But this is still wrong though, since Optional is a struct, there shouldn't be any reason to null check it.

You should instead do `rtcRegion.IfPresent(e => e?.Id)`, this will generate either an empty Optional<string> (no change), an Optional<string> with a value of `null` (change to automatic voice region), or an Optional<string> with a region value (change to X voice region)